src/redux/actions/authActions/successSignInAction.ts

```diff
@@ -1,19 +1,19 @@
-import { TodoActionTypes } from './index';
+import { AuthActionsTypes } from './index';
 import { FirebaseAuthTypes } from '@react-native-firebase/auth';
 
-export const signInAction = (payload: SignInPayload) => {
+export const successSignInAction = (payload: SignInPayload) => {
   return {
-    type: TodoActionTypes.SIGN_IN,
+    type: AuthActionsTypes.SIGN_IN,
     payload,
   };
 };
 
 export type SignInPayload = {
   userToken: FirebaseAuthTypes.User['uid'] | null;
-  user: FirebaseAuthTypes.UserCredential['user'];
+  user: FirebaseAuthTypes.UserCredential['user'] | undefined;
```

Comment:
you can write `user?: FirebaseAuthTypes.UserCredential['user']` and remove `undefined `

Reply:
i fix it